src/methods/search.js

```diff
@@ -0,0 +1,61 @@
+/* @flow */
+
+export default function search(): Object {
+  return {
+    all: (keyword, page = 1)  => {
```

Comment:
Should we provide an empty string fallback for `keyword`? or does the API throw a meaningful error when keyword is missing?